src/azure-cli-core/azure/cli/core/commands/command_operation.py

```diff
@@ -72,7 +72,11 @@ def load_getter_op_arguments(self, getter_op, cmd_args=None):
     def apply_doc_string(self, handler):
         return self.ctx._apply_doc_string(handler, self.kwargs)  # pylint: disable=protected-access
 
-    def load_op_description(self, handler):
+    def load_op_description(self, handler=None):
+        if handler is None:
+            def handler():
+                """"""
```

Comment:
```suggestion
```

Reply:
> 

If remove this line, the __doc__ property of handler cannot be updated in apply_doc_string

Reply:
I will add comments in code later.

Reply:
How about adding some text as placeholder?

Reply:
> How about adding some text as a placeholder?

It's a good idea. But the placeholder text will be the default description of that handler.